doc/telemetry-dev-env/Readme.md

```diff
@@ -0,0 +1,46 @@
+Build the image:
```

Comment:
It might be helpful to emphasize that this command should run under `doc/telemetry-dev-env`